torch_frame/data/multi_nested_tensor.py

```diff
@@ -199,6 +217,23 @@ def device(self) -> torch.device:
 
     def size(self, dim: int) -> int:
         r"""Dimension of the :class:`torch_frame.data.MultiNestedTensor`
+
+        Example:
+            >>> import torch
+            >>> tensor_mat = [
+            ...    [torch.tensor([1, 2]), torch.tensor([3])],
+            ...    [torch.tensor([4]), torch.tensor([5, 6, 7])],
+            ...    [torch.tensor([8, 9]), torch.tensor([10])],
+            ... ]
+            >>> out = MultiNestedTensor.from_tensor_mat(tensor_mat)
+            >>> out.size(0)
+            3
+            >>> out.size(1)
+            2
+            >>> out.size(2)
+            Traceback (most recent call last):
+            File "<stdin>", line 1, in <module>
+            ValueError: MultiNestedTensor does not have a fixed length on the third dimension.  # noqa
```

Comment:
I don't know if this example is necessary though. It seems pretty obvious. Also if we are to keep it. I think it's better to move it to the class's  doc-string rather than put it in a function's doc-string.

Reply:
+1.

Reply:
As a user, I'm totally fine with the docs having some verbose examples, so I'll just move it to the class docsring :) Feel free to remove the example if it's too dirty to include in our API reference :)